### PR TITLE
Bump to 4.3.14

### PR DESCRIPTION
**Pull request by basnijholt**
Fixes https://github.com/conda-forge/conda-feedstock/issues/16

I am frequently encountering this error https://github.com/conda/conda/issues/3494 

It seems to have been solved in 4.3, therefore I created this PR.

### CODE REVIEW

**Comment by jakirkham** [comment]
Are there new requirements needed?

**Comment by jakirkham** [comment]
Am ok with this, but should note we will need to do a bit of pinning of `conda` throughout conda-forge to make sure things work. That will take a little bit of time.

**Comment by jjhelmus** [comment]
> Are there new requirements needed?

I believe `pyopenssl >= 16.2` should be added as a run requirement and the minimum version of `requests` should be updated to `>= 2.12.4`.  These are based upon the [meta.yaml file in anaconda-recipes](https://github.com/ContinuumIO/anaconda-recipes/blob/master/conda-4.3/meta.yaml).

File: recipe/meta.yaml [hunk 1]
```diff
@@ -49,8 +49,9 @@ requirements:
     - enum34                 # [py<34]
     - menuinst               # [win]
     - pycosat >=0.6.1
-    - requests >=2.5.3
+    - requests >=2.12.4
     - ruamel_yaml >=0.11.14
+    - pyopenssl >= 16.2
```
Comment (jakirkham) [inline recipe/meta.yaml]: No space after the `>=`.

**Comment by jakirkham** [comment]
Please rebase.

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipe/meta.yaml]: FYI this is causing the build failures.

**Comment by basnijholt** [comment]
I have updated this PR to 4.3.14. 

I do not understand what is going wrong with the builds.

**Comment by marscher** [comment]
Maybe this warning message should be taken into account?
WARNING: Your current install method for conda only supports conda
as a python library.  You are not installing a conda executable command
or activate/deactivate commands.  If your intention is to install conda
as a standalone application, currently supported install methods include
the Anaconda installer and the miniconda installer.  If you'd still like
for setup.py to create entry points for you, use `utils/setup-testing.py`.

**Comment by marscher** [comment]
By having a look on how conda is built by Continuum Analytics, they provide custom build scripts to link the activate/deactivate scripts. I've attached the recipe.tar from the linux binary, so you can copy the relevant code @basnijholt. BTW thanks a lot for your effort to upgrade conda 👍 

[build_scripts.zip](https://github.com/conda-forge/conda-feedstock/files/862346/build_scripts.zip)

**Comment by RafalSkolasinski** [comment]
@basnijholt Bump to 4.3.16?

**Pull request merged**